src/improvements.js

```diff
@@ -0,0 +1,59 @@
+//exercise 1
+
+function calculateAverage(numbers) {
+  return numbers.reduce((acc, num) => acc + num, 0) / numbers.length;
+}
+
+//Exercise 2
+
+function greet(name) {
+  return name ? `Hello, ${name} !` : "Hello!";
```

Comment:
I suggest  that here you stick to using backticks for the else case(when there is no name)